packages/x-adapter-platform/src/schemas/requests/ai/suggestions-search-request.schema.ts

```diff
@@ -11,10 +11,14 @@ export const aiSuggestionsSearchRequestSchema = createMutableSchema<
   AiSuggestionsSearchRequest,
   PlatformAiSuggestionsSearchRequest
 >({
-  context: ({ extraParams }) => ({
-    lang: (extraParams?.lang as string | undefined) ?? '',
-    instance: extraParams?.instance as string | undefined,
-    filters: extraParams?.filters as Record<string, unknown> | undefined,
-  }),
+  context: ({ extraParams, origin }) => {
```

Comment:
Should we do the same for the `suggestions-request.schema`? 

I tested it, and it seems like propagating the origin to tagging response.

<img width="510" height="229" alt="Image" src="https://github.com/user-attachments/assets/49ca433d-dc06-4f4f-9a96-0cbb0ceafe58" />

<img width="918" height="328" alt="Image" src="https://github.com/user-attachments/assets/456e72dc-bfb2-467c-a382-e6780dca26b0" />